web/gds-admin-ui/src/utils/index.js

```diff
@@ -43,7 +43,7 @@ function formatDisplayedData(target) {
 
 const convertCountsToPercentages = (data) => {
     const total = Object.values(data).reduce((acc, x) => acc + x);
-    return Object.fromEntries(Object.entries(data).map(([k, v]) => [k, v / total]));
+    return Object.fromEntries(Object.entries(data).map(([k, v]) => [k, (v / total).toFixed(1)]));
```

Comment:
You'll need to update this to:

```
return Object.fromEntries(Object.entries(data).map(([k, v]) => [k, ((v / total)* 100.0).toFixed(1)]));
```

In order to get the `%` rather than the ratio.

Reply:
However, the donut chart might require the ratios so you might have to do this computation second.